tests/rptest/tests/pandaproxy_test.py

```diff
@@ -911,13 +927,28 @@ def test_consumer_group_json_v2(self):
 
         # Fetch from a consumer
         self.logger.info(f"Consumer fetch")
-        cf_res = c0.fetch(headers=HTTP_CONSUMER_FETCH_JSON_V2_HEADERS)
-        assert cf_res.status_code == requests.codes.ok
-        fetch_result = cf_res.json()
+        fetch_result = []
+
+        def do_fetch():
```

Comment:
This looks like the same `do_fetch` as above. Can you make a single function?

Reply:
Sure.